src/inputs.js

```diff
@@ -0,0 +1,11 @@
+const core = require("@actions/core");
+
+module.exports = {
+  getStackYaml: () => {
```

Comment:
Super nitpicky and not at all ubiquitous, I've noticed some in the frontend codebase moving to "expression-style" over "statement-style".

Meaning something like
``` javascript
getStackYaml: () => core.getInput("stack-yaml", { required: true })
```

Reply:
Since we're typing Y.A.M.L. over Y.M.L. we've gotta get those keystrokes back however we can, am I right?
:postal_horn: :clown_face:

Reply:
Do you just mean dropping the `{ }` for a one-line function? I'm down with that!

Reply:
Yep!